test/categorical_algebra/CSets.jl

```diff
@@ -114,13 +114,17 @@ g, h = path_graph(Graph, 4), cycle_graph(Graph, 2)
 @test components(α′′) == components(α)
 
 # Naturality.
+d = get_unnaturalities(α)
+@test [collect(d[a]) for a in keys(d)] == [[],[]]
 @test is_natural(α)
 β = CSetTransformation((V=[1,2,1,2], E=[1,1,1]), g, h)
+d = get_unnaturalities(β)
+@test [collect(d[a]) for a in keys(d)] == [[(2,1,2)],[(2,2,1)]]
```

Comment:
This test is failing some versions of Julia because the order of iteration of dictionary keys is not guaranteed to be stable between different versions. This should be:

```julia
sort([collect(v) for v in values(d)]) == [[(2,1,2)],[(2,2,1)]]
```